Dockerfile.WorkerSetupJob

```diff
@@ -0,0 +1,19 @@
+FROM registry.access.redhat.com/ubi8/python-39
+
+USER 0
+
+COPY resources/ansible/requirements.yml /tmp/requirements.yml
+
+RUN dnf install skopeo -y && \
+    pip install --upgrade pip && \
+    pip install openshift && \
+    pip install jmespath && \
+    pip install netaddr && \
+    pip install ansible==2.9.* && \
+    curl -L https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz | \
```

Comment:
in a disconnected env this should be replaced, doesn't it?

Reply:
No, we build this image ourselves and push it to our container registry.